exporter/awsxrayexporter/internal/translator/http.go

```diff
@@ -54,7 +54,14 @@ func makeHTTP(span pdata.Span) (map[string]pdata.AttributeValue, *awsxray.HTTPDa
 			info.Request.UserAgent = awsxray.String(value.StringVal())
 			hasHTTP = true
 		case conventions.AttributeHTTPStatusCode:
-			info.Response.Status = aws.Int64(value.IntVal())
+			statusCode := value.IntVal()
+			if value.Type() == pdata.AttributeValueTypeString {
+				statusCodeStr := value.StringVal()
+				v, _ := strconv.ParseInt(statusCodeStr, 10, 64)
+				statusCode = v
```

Comment:
Why ignoring the error?

Reply:
Originally I was logging it, but I noticed a few errors where errors were ignored, and since there is no logging in this package, I decided to ignore it. One option is to log it using `zap.L()` but nothing sets loggers up on the global space nor are they really used anywhere except the processors (and I think a couple other areas). So because of this, I thought ignoring may be the best option. However, if we want to log it using `zap.L()` it'll be a noop. Another option is to pass a logger in, but that would require passing it all the way down, which the exporters do not get. So that would need to change.

Reply:
We definitely pass a Logger to all the components (receivers, processors, exporters), see https://github.com/open-telemetry/opentelemetry-collector/blob/main/component/receiver.go#L120 (equivalent for all, we send a create settings which includes telemetry settings), see https://github.com/open-telemetry/opentelemetry-collector/blob/main/component/telemetry.go#L23.

Reply:
Ah! Don't know how I missed the loggers in the exporters. I swear I tried grepping a little bit ago, but they are there. Let me log that error.

Reply:
@bogdandrutu Alright, updated and no I am passing the logger down to the internal package and logging the error. Please take a look when you get a chance!

Reply:
So there looks to be already a parsing helper that can be set in the config. I'll go ahead and set that, and things should be parsed correctly